packages/bootstrap-4/src/FieldTemplate/WrapIfAdditionalTemplate.tsx

```diff
@@ -34,8 +24,9 @@ const WrapIfAdditional = ({
   schema,
   uiSchema,
   registry,
-}: WrapIfAdditionalProps) => {
-  const { RemoveButton } = registry.templates.ButtonTemplates;
+}: WrapIfAdditionalTemplateProps) => {
+  const uiOptions = getUiOptions(uiSchema);
+  const { RemoveButton } = getTemplate("ButtonTemplates", registry, uiOptions);
```

Comment:
Consider restoring this since we don't need the extra overhead of the call as we can't override them in `uiOptions`. You could add this comment which is in many of the other places (but this one)
```
  // Button templates are not overridden in the uiSchema
```